scripts/cibuild

```diff
@@ -54,7 +54,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
                   -f "${DIR}/../docker-compose.yml" \
                   -f "${DIR}/../docker-compose.airflow.yml" \
                   -f "${DIR}/../docker-compose.test.yml" \
-                  build airflow
+                  build airflow airflow-worker
```

Comment:
On third thought, there's apparently a conflict that makes it so that I need to do this --

Running anything from the batch jar depends on having the Rollbar token available. Running the airflow test suite depends on being able to initialize boto. When I threw the `.env` file into the airflow container from `docker-compose.test.yml`, boto couldn't initialize because the `raster-foundry` profile wasn't available in the airflow container.